regimes/my/tax_categories.go

```diff
@@ -0,0 +1,111 @@
+package my
+
+import (
+	"github.com/invopop/gobl/cal"
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/num"
+	"github.com/invopop/gobl/tax"
+)
+
+var taxCategories = []*tax.CategoryDef{
+	//
+	// Sales Tax (part of SST)
+	//
+	{
+		Code: "sales-tax", // Custom subcategory under SST
+		Name: i18n.String{
+			i18n.EN: "Sales Tax",
+		},
+		Title: i18n.String{
+			i18n.EN: "Sales and Service Tax (Sales Portion)",
+		},
+		Sources: []*cbc.Source{
+			{
+				Title: i18n.String{
+					i18n.EN: "Royal Malaysian Customs Department - Sales and Service Tax",
+				},
+				URL: "https://mysst.customs.gov.my/",
+			},
+		},
+		Rates: []*tax.RateDef{
+			{
+				Key: "standard-5",
```

Comment:
The objective with the keys is to avoid having to define a specific percentage so that if there are modifications on a specific date, percentages will be updated automatically. Something like `reduced` might make sense here.